### PR TITLE
Update package.json with version and correct endpoint

### PR DESCRIPTION
**Pull request by zriyansh**

### CODE REVIEW

**Comment by zriyansh** [comment]
cc - @makeavish @pranay01 @pranay01

File: package.json [hunk 1]
```diff
@@ -10,13 +10,13 @@
   },
   "scripts": {
     "start": "node -r ./tracing.js index.js",
-    "start:local": "OTEL_EXPORTER_OTLP_ENDPOINT=\"http://localhost:4317\" OTEL_RESOURCE_ATTRIBUTES=service.name=node-js-sample-app node -r ./tracing.js index.js"
+    "start:local": "OTEL_EXPORTER_OTLP_ENDPOINT=\"http://stagingapp.signoz.io:4318/v1/traces\" OTEL_RESOURCE_ATTRIBUTES=service.name=node-js-sample-app node -r ./tracing.js index.js"
```
Comment (prashant-shahi) [inline package.json]: can you revert this to `localhost:4317`?

**Review by makeavish — CHANGES_REQUESTED** [review]
Don't push staging server endpoint.

File: package.json (hunk 1 above)
Comment (zriyansh) [reply package.json]: localhost:4317 or 4318? the tracing.js file of this app shows 4318 @prashant-shahi

Comment (zriyansh) [reply package.json]: okay, reverted back to localhost:4317

**Review by makeavish — APPROVED** [review]

**Pull request merged**